docs/reference/migration/migrate_7_0/search.asciidoc

```diff
@@ -18,3 +18,7 @@ PUT /_cluster/settings
 --------------------------------------------------
 // CONSOLE
 
+=== Scroll search will response `bad request` if request is invalid
```

Comment:
I think we can skip the java exception names here given that this guide is read by all our users, also the ones that are not familiar with Java. Let's focus on response codes only. Also I would make the title shorter:

```
=== `_search/scroll` returns `400` for invalid requests

The `/_search/scroll` endpoint returns `400 - Bad request` when the request invalid, while it would previously return `500 - Internal Server Error` in such case.

```